packages/plugin-core/src/utils/StartupUtils.ts

```diff
@@ -35,8 +35,115 @@ import { SurveyUtils } from "../survey";
 import { VSCodeUtils } from "../vsCodeUtils";
 import { AnalyticsUtils } from "./analytics";
 import { ConfigMigrationUtils } from "./ConfigMigration";
+import semver from "semver";
 
 export class StartupUtils {
+  static shouldShowManualUpgradeMessage({
+    previousWorkspaceVersion,
+    currentVersion,
+  }: {
+    previousWorkspaceVersion: string;
+    currentVersion: string;
+  }) {
+    const workspaceInstallStatus = VSCodeUtils.getInstallStatusForWorkspace({
+      previousWorkspaceVersion,
+      currentVersion,
+    });
+
+    return (
+      workspaceInstallStatus === InstallStatus.UPGRADED &&
+      semver.lte(previousWorkspaceVersion, "0.63.0")
+    );
+  }
+
+  static showManualUpgradeMessage() {
+    const SHOW_ME_HOW = "Show Me How";
+    const MESSAGE =
+      "You are upgrading from a really old version of Dendron. Please follow the instructions to manually migrate your configurations.";
+    vscode.window
+      .showInformationMessage(MESSAGE, SHOW_ME_HOW)
+      .then(async (resp) => {
+        if (resp === SHOW_ME_HOW) {
+          AnalyticsUtils.track(MigrationEvents.ManualUpgradeMessageConfirm, {
+            status: ConfirmStatus.accepted,
+          });
+
+          const content = [
+            "# Instructions for Upgrade",
+            "",
+            "You are upgrading from a version that is below 0.63.0, which we do not support direct automatic migration.",
+            "",
+            "You can either perform a stepped upgrade, or manually move the migrated configurations to the correct location.",
```

Comment:
I don't think most folks will be familiar with what a "stepped upgrade" means. would rephrase - you can either manually update the configuration or update to a compatible older version before upgrading to the current version

also - since these instructions can change and have multiple steps, would be better to link to the dendron wiki vs trying to write it out in this markdown message.

Reply:
If we are going to link to a note in the wiki, should we just ditch the webview altogether and just open the link when the toast message was accepted?

Reply:
yep